tests/test_data_import.py

```diff
@@ -0,0 +1,435 @@
+    output_df = read_ornlabuse(cell_id, file_path)
+    result_df_cols = [
+        "test_time",
+        "axial_d",
+        "v",
+        "axial_f",
+        "temp_1",
+        "temp_2",
+        "temp_3",
+        "temp_4",
+        "temp_5",
+        "temp_6",
+        "cell_id",
+    ]
+    result_df = [
+        {
+            "test_time": 0.000,
+            "axial_d": 0,
+            "v": 4.128,
+            "axial_f": 0,
+            "temp_1": 0.00,
+            "temp_2": 0.00,
+            "temp_3": 0.00,
+            "temp_4": 0.00,
+            "temp_5": 0.00,
+            "temp_6": 0.00,
+            "cell_id": cell_id,
+        },
+        {
+            "test_time": 0.046,
+            "axial_d": 0,
+            "v": 4.137,
+            "axial_f": 1,
+            "temp_1": 0.00,
+            "temp_2": 0.00,
+            "temp_3": 0.00,
+            "temp_4": 0.00,
+            "temp_5": 0.00,
+            "temp_6": 0.00,
+            "cell_id": cell_id,
+        },
+        {
+            "test_time": 0.000,
+            "axial_d": 0,
+            "v": 0,
+            "axial_f": 0,
+            "temp_1": 22.27,
+            "temp_2": 22.12,
+            "temp_3": 22.44,
+            "temp_4": 22.03,
+            "temp_5": 22.08,
+            "temp_6": 22.58,
+            "cell_id": cell_id,
+        },
+        {
+            "test_time": 0.500,
+            "axial_d": 0,
+            "v": 0,
+            "axial_f": 0,
+            "temp_1": 22.23,
+            "temp_2": 22.03,
+            "temp_3": 22.56,
+            "temp_4": 22.07,
+            "temp_5": 22.12,
+            "temp_6": 22.62,
+            "cell_id": cell_id,
+        },
+    ]
+    result_pd_df = pd.DataFrame(data=result_df, columns=result_df_cols)
+    df_print(output_df, result_pd_df)
+
+    assert output_df.equals(result_pd_df)
+
+
+def test_read_snlabuse():
+    cell_id = "S2Abuse"
+    file_path = rawTestDataPath + "abuse-snl/"
+    output_df = read_snlabuse(cell_id, file_path)
+    result_df_cols = [
+        "test_time",
+        "axial_d",
+        "axial_f",
+        "v",
+        "temp_1",
+        "temp_2",
+        "temp_3",
+        "temp_4",
+        "temp_5",
+        "temp_6",
+        "cell_id",
+    ]
+    result_df = [
+        {
+            "test_time": 1.06,
+            "axial_d": -0.055,
+            "axial_f": 4.9,
+            "v": 4.132,
+            "temp_1": 20.3,
+            "temp_2": 20,
+            "temp_3": 20.2,
+            "temp_4": 20.1,
+            "temp_5": 20.2,
+            "temp_6": 20.1,
+            "cell_id": cell_id,
+        },
+        {
+            "test_time": 2.06,
+            "axial_d": -0.085,
+            "axial_f": 3.503,
+            "v": 4.132,
+            "temp_1": 20.4,
+            "temp_2": 20,
+            "temp_3": 20.2,
+            "temp_4": 20.1,
+            "temp_5": 20.2,
+            "temp_6": 20.1,
+            "cell_id": cell_id,
+        },
+    ]
```

Comment:
This test is failing for me because the output dataframe from `read_snlabuse` does not match your test dataframe. In `data_import.py` the output structure uses the following values instead:

```suggestion
    result_df_cols = [
        "test_time",
        "axial_d",
        "axial_f",
        "v",
        "pos_terminal_temperature",
        "neg_terminal_temperature",
        "left_bottom_temperature",
        "right_bottom_temperature",
        "above_punch_temperature",
        "below_punch_temperature",
        "cell_id",
    ]
    result_df = [
        {
            "test_time": 1.06,
            "axial_d": -0.055,
            "axial_f": 4.9,
            "v": 4.132,
            "pos_terminal_temperature": 20.3,
            "neg_terminal_temperature": 20,
            "left_bottom_temperature": 20.2,
            "right_bottom_temperature": 20.1,
            "above_punch_temperature": 20.2,
            "below_punch_temperature": 20.1,
            "cell_id": cell_id,
        },
        {
            "test_time": 2.06,
            "axial_d": -0.085,
            "axial_f": 3.503,
            "v": 4.132,
            "pos_terminal_temperature": 20.4,
            "neg_terminal_temperature": 20,
            "left_bottom_temperature": 20.2,
            "right_bottom_temperature": 20.1,
            "above_punch_temperature": 20.2,
            "below_punch_temperature": 20.1,
            "cell_id": cell_id,
        },
    ]
```